features/step_definitions/hangout_steps.rb

```diff
@@ -117,10 +118,26 @@
   event = Event.find_by_name(event_name)
   visit event_path(event)
   expect(page).to have_link('Join now', href: @hangout_url)
-  time = Time.parse(@jump_date) + event.duration.minutes - 1.minute
+  time = Time.parse(@jump_date) + event.duration.minutes - 10.minutes
   Delorean.time_travel_to(time)
   visit event_path(event)
   expect(page).to have_link('Join now', href: @hangout_url)
+  time = Time.parse(@jump_date) + event.duration.minutes + 10.minutes
+  Delorean.time_travel_to(time)
+  visit event_path(event)
+  expect(page).not_to have_link('Join now')
+end
+
+When(/^I start the "([^"]*)" event$/) do |event_name|
+  event = Event.find_by_name(event_name)
+  visit event_path(event)
+  find('#liveHOA-placeholder').click
```

Comment:
@tansaku, and this doesn't happen when clicking on "Start Hangout", right? This is complicated to figure out, as the hangout doesn't start in the development environment.
Known, that event can be started by filling in the `hangout_url` field.
The part of the logic in the `events/show.html.erb`, that increasing number of moving parts.
Since you are replacing the conferencing platform, apparently, this test should be reconsidered.
Essentially, the first approach of @lokesh solves the task. The intention was to make scenario clearer.